### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by rosemarym2**

### CODE REVIEW

**Review by grovrac09 — APPROVED** [review]

File: src/components/index.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  grid-template-columns: repeat(3, 33.3%);
+  grid-template-rows: repeat(4, 33.3%);
+  padding: 35px;
+}
+
+.create {
+  text-align: center;
+}
+
+.Img {
```
Comment (Ju1ija) [inline src/components/index.css:27]: can we make it a lower case?

File: src/components/home.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import'./index.css';
+
+
+export const Home = (props) =>{
+  return (
+    <div className = "text">
+    <h1>Pick A List</h1>
+    <div>
```
Comment (Ju1ija) [inline src/components/home.js:9]: doesn't need to be in a div

File: src/components/home.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    <MainArticle 
+    title = "Categories"
+    />
+    </div>
+    <div className="home">
+    <Article  
+    Img = "https://res.cloudinary.com/cn-project/image/upload/v1641481525/pana/Winter_solstice-pana_x4wbwx.png"
```
Comment (Ju1ija) [inline src/components/home.js:16]: can we change it to something lower case? "image" if we don't want to use "img"

File: src/components/home.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <MainArticle 
+    title = "Categories"
+    />
+    </div>
+    <div className="home">
+    <Article  
+    Img = "https://res.cloudinary.com/cn-project/image/upload/v1641481525/pana/Winter_solstice-pana_x4wbwx.png" 
+    alt= "Animated picture of the world globe"
+    title = "Travel"
+    // category= "Travel"
```
Comment (Ju1ija) [inline src/components/home.js:19]: do we need to keep these comments?

File: src/components/home.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    title = "New List"
+    // category= "Create your own"
+    /> 
+    </div>   
+    </div>
+  )
+}
+
+const MainArticle = (props) =>{
+  return (<div className ="MainArticle">
```
Comment (Ju1ija) [inline src/components/home.js:56]: can we use camelCase for class names? as in, mainArticle or main-article, not MainArticle

File: src/components/home.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  <img src = {props.image1} className= "mainImg"/>
+  <h2>{props.title}</h2>
+  </div>
+  )}
+
+const Article = (props) =>{
+  return (<div className ="Article">
```
Comment (Ju1ija) [inline src/components/home.js:63]: same here pls, class names camelCase :)

File: src/components/home.js [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    // category= "Create your own"
+    /> 
+    </div>   
+    </div>
+  )
+}
+
+const MainArticle = (props) =>{
+  return (<div className ="MainArticle">
+  <img src = {props.image1} className= "mainImg"/>
```
Comment (Ju1ija) [inline src/components/home.js:57]: are we using this anywhere? If not, can we remove it?

File: src/components/home.js (hunk 2 above)
Comment (rosemarym2) [reply src/components/home.js:9]: Yes this was needed for the return to work

File: src/components/home.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  <h2>{props.title}</h2>
+  </div>
+  )}
+
+const Article = (props) =>{
+  return (<div className ="Article">
+  <img src = {props.Img} className= "Img" />
```
Comment (Ju1ija) [inline src/components/home.js:64]: camelCase for both Img

File: src/components/home.js (hunk 3 above)
Comment (rosemarym2) [reply src/components/home.js:16]: We can but it works as it is and it helps differentiate between the main pages images in css

File: src/components/home.js (hunk 4 above)
Comment (rosemarym2) [reply src/components/home.js:19]: The comments were placeholders for us to discuss what else needs to go into this section together but can be removed once they have been decided on

File: src/components/home.js (hunk 5 above)
Comment (rosemarym2) [reply src/components/home.js:56]: yes we can change this

File: src/components/home.js (hunk 7 above)
Comment (rosemarym2) [reply src/components/home.js:57]: Again, this is a placeholder for us to discuss as a group

File: src/components/home.js (hunk 8 above)
Comment (rosemarym2) [reply src/components/home.js:64]: so Img is the code for the name within the React tag and img is the html code

**Review by Ju1ija — APPROVED** [review]

**Pull request merged**